libsolidity/interface/Utils.h

```diff
@@ -38,6 +38,9 @@ struct UnimplementedFeatureError;
 #define solAssert(CONDITION, DESCRIPTION) \
 	assertThrow(CONDITION, ::dev::solidity::InternalCompilerError, DESCRIPTION)
 
+#define astAssert(CONDITION, DESCRIPTION) \
+	assertThrow(CONDITION, ::dev::solidity::InternalCompilerError, DESCRIPTION)
```

Comment:
Thought it would be nicer to have proper error reporting, such as an *AST analyzer/validator* class which uses the errorReporter.